priv/repo/seeds.exs

```diff
@@ -1,6 +1,51 @@
-alias Aot.ProjectActions
+alias Aot.{ProjectActions, NodeActions, SensorActions, ObservationActions}
 
-{:ok, chi} =
-  ProjectActions.create name: "Chicago",
-    archive_url: "https://www.mcs.anl.gov/research/projects/waggle/downloads/datasets/AoT_Chicago.complete.latest.tar",
-    recent_url: "https://www.mcs.anl.gov/research/projects/waggle/downloads/datasets/AoT_Chicago.complete.recent.tar"
+try do
+  {:ok, _} =
+    ProjectActions.create name: "Chicago",
+      archive_url: "https://www.mcs.anl.gov/research/projects/waggle/downloads/datasets/AoT_Chicago.complete.latest.tar",
+      recent_url: "https://www.mcs.anl.gov/research/projects/waggle/downloads/datasets/AoT_Chicago.complete.recent.tar"
+rescue
+  error in Ecto.ConstraintError ->
+    IO.inspect(error)
+end
+
+try do
```

Comment:
You could call the function to load it now. This seed data doesn't look like the real stuff and isn't very helpful.